### ISSUE TITLE
Search string doesn't allow search for System

### ISSUE THREAD

**Issue by zpaolo11x** [mickelson/attract#581]
While fe.game_info keeps Info.System information, the search rule can't use "System equals ..." structure but only "Emulator". It would be great if one could search for System too

**Comment by mickelson** [comment]
Yes I agree.  Unfortunately there does not appear to be an easy fix given how things are currently implemented...
